Project/Assets/ML-Agents/Examples/Walker/Prefabs/Ragdoll.meta

```diff
@@ -0,0 +1,8 @@
+fileFormatVersion: 2
```

Comment:
Same comment here, I do not think creating a new prefab is worth it if only one checkbox is changed.

Reply:
This is the base prefab for all of the ragdoll prefab variants. All other Walker prefabs inherit the changes made to this prefab. It's useful when updating the different environments.

Reply:
Is it good practice to have multiple prefabs that are almost identical?

Reply:
if everything used the same prefab, anytime that prefab was updated it would update all in every environment. it would change the Behavior Name, Target type, walk settings...etc. Each uses slightly different settings.